contrib/format-hdf5/src/main/java/org/apache/drill/exec/store/hdf5/HDF5BatchReader.java

```diff
@@ -946,7 +990,11 @@ private void writeAttributes(TupleWriter rowWriter, HDF5DrillMetadata record) {
           writeLongColumn(mapWriter, key, (Long) attrib.getValue());
           break;
         case INT:
-          writeIntColumn(mapWriter, key, (Integer) attrib.getValue());
+          try {
+            writeIntColumn(mapWriter, key, (Integer) attrib.getValue());
+          } catch (Exception e) {
+            logger.warn("{} {}", key, attrib);
```

Comment:
Is this the right solution? What would cause the exception? If reading 100M rows, do we want to emit 100M warnings into the log? Will the user understand why the value in some column is null?

Should we be more selective: catching the (one) exception we want to ignore and failing for all others?

Reply:
I'm not sure why this try/catch block was here but I removed it entirely.